.travis.yml

```diff
@@ -21,7 +21,8 @@ before_script:
   - ./configure --enable-ccache
 script:
   - make tidy
-  - make rustc-stage1 -j4
+  - make -j4
```

Comment:
Ah and to clarify `make check` in parallel has been fixed recently and I haven't run into any problems with it recently as well (plus this is a good way to test it). Building the unit tests in parallel may have a definite win.